rosdep/python.yaml

```diff
@@ -354,6 +354,13 @@ pyqt5-dev-tools:
   fedora: [python-qt5-devel]
   gentoo: [dev-python/PyQt5]
   ubuntu: [pyqt5-dev-tools]
+pyquaternion-pip:
```

Comment:
Please use the key name `python3-pyquaternion-pip`

https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1